src/Controller/AdminController.php

```diff
@@ -450,8 +450,8 @@ private function getFilteredClassDefinitionList(Request $request): ClassDefiniti
             $allowedClassIds = [];
             foreach ($this->defaultGridClasses as $allowedClass) {
                 $classNamespace = 'Pimcore\\Model\\DataObject\\';
-                $allowedClass = explode('\\', $allowedClass);
-                $allowedClassFull = $classNamespace . array_pop($allowedClass);
+                $allowedClassArr = explode('\\', $allowedClass);
```

Comment:
Am I missing something here or is it just the variable which has been renamed?

Reply:
Before, $allowedClass was an array and hence it was throwing the error. So, I just added a new variable for the array, so that $allowedClass will stay as such.